examples/run/cpu/throughput_seq_example.cpp

```diff
@@ -0,0 +1,149 @@
+    }
+    // Seed random number generator
+    std::srand(std::time(0));
+
+    // Dummy count uses output of tp algorithm to ensure the compiler
+    // optimisations don't skip this step
+    std::size_t dummy_count = 0;
+
+    // Loop over events
+    for (int i = 0; i < i_cfg.processed_events; ++i) {
+
+        const int event = std::rand() % i_cfg.loaded_events;
+
+        {
+
+            traccc::performance::timer t("Events processing", elapsedTimes);
+
+            traccc::cell_container_types::host& cells_per_event =
+                cells_event_vec[event];
+
+            /*-------------------
+                Clusterization
+              -------------------*/
+
+            auto measurements_per_event = ca(cells_per_event);
+
+            /*------------------------
+                Spacepoint formation
+              ------------------------*/
+
+            auto spacepoints_per_event = sf(measurements_per_event);
+
+            /*-----------------------
+              Seeding algorithm
+              -----------------------*/
+
+            auto seeds = sa(spacepoints_per_event);
+
+            /*----------------------------
+              Track params estimation
+              ----------------------------*/
+
+            dummy_count += tp(spacepoints_per_event, seeds).size();
+        }
+    }
+    std::cout << "dummy_count = " << dummy_count << std::endl;
+
+    std::cout << "==> Statistics ... " << std::endl;
+    std::cout << "- Processed " << i_cfg.processed_events << " events from "
+              << i_cfg.loaded_events << " loaded events." << std::endl;
+
+    std::cout << "File reading throughput (ms/event) = "
+              << elapsedTimes.get_time("File reading").count() * 1.e-6 /
+                     (double)i_cfg.loaded_events
+              << std::endl;
+    std::cout << "Processing throughput   (ms/event) = "
+              << elapsedTimes.get_time("Events processing").count() * 1.e-6 /
+                     (double)i_cfg.processed_events
+              << std::endl;
+
+    return 0;
+}
+
+// The main routine
+//
+int main(int argc, char* argv[]) {
+    // Set up the program options
+    po::options_description desc("Allowed options");
+
+    // Add options
+    desc.add_options()("help,h", "Give some help with the program's options");
+    traccc::throughput_full_tracking_input_config batch_full_tracking_input_cfg(
+        desc);
+
+    po::variables_map vm;
+    po::store(po::parse_command_line(argc, argv, desc), vm);
+
+    // Check errors
+    traccc::handle_argument_errors(vm, desc);
+
+    // Read options
+    batch_full_tracking_input_cfg.read(vm);
+
+    std::cout << "Running " << argv[0] << " "
+              << batch_full_tracking_input_cfg.detector_file << " "
+              << batch_full_tracking_input_cfg.input_directory << " "
+              << batch_full_tracking_input_cfg.loaded_events << " "
+              << batch_full_tracking_input_cfg.processed_events << std::endl;
```

Comment:
I never liked these printouts. :thinking: Let's come up with a better printout for the new applications at least.

How about you define an

```c++
std::ostream& operator<<(std::ostream&, const traccc::throughput_full_tracking_input_options&);
```

"printout operator". And then we just rely on that to produce some meaningful output at the start of the applications.